monai/metrics/compute_meandice.py

```diff
@@ -9,55 +9,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import warnings
+
 import torch
 
 from monai.networks.utils import one_hot
 
 
 def compute_meandice(y_pred,
                      y,
-                     include_background=False,
+                     include_background=True,
                      to_onehot_y=True,
-                     mutually_exclusive=True,
+                     mutually_exclusive=False,
                      add_sigmoid=False,
-                     logit_thresh=None):
+                     logit_thresh=0.5):
     """Computes dice score metric from full size Tensor and collects average.
 
     Args:
         y_pred (torch.Tensor): input data to compute, typical segmentation model output.
-                               it must be One-Hot format and first dim is batch, example shape: [16, 3, 32, 32].
+            it must be One-Hot format and first dim is batch, example shape: [16, 3, 32, 32].
         y (torch.Tensor): ground truth to compute mean dice metric, the first dim is batch.
-                          example shape: [16, 3, 32, 32] for 3-class one-hot labels.
-                          alternative shape: [16, 1, 32, 32] and set `to_onehot_y=True` to convert it into [16, 3, 32, 32].
-        include_background (Bool): whether to skip dice computation on the first channel of the predicted output.
-        to_onehot_y (Bool): whether to convert `y` into the one-hot format.
+            example shape: [16, 1, 32, 32] will be converted into [16, 3, 32, 32].
+            alternative shape: [16, 3, 32, 32] and set `to_onehot_y=False` to use 3-class labels directly.
+        include_background (Bool): whether to skip Dice computation on the first channel of
+            the predicted output. Defaults to True.
+        to_onehot_y (Bool): whether to convert `y` into the one-hot format. Defaults to True.
         mutually_exclusive (Bool): if True, `y_pred` will be converted into a binary matrix using
-            a combination of argmax and to_onehot.
-        add_sigmoid (Bool): whether to add sigmoid function to y_pred before computation.
-        logit_thresh (Float): the threshold value used to convert `y_pred` into a binary matrix.
+            a combination of argmax and to_onehot.  Defaults to False.
```

Comment:
Best for mutually_exclusive to default to same value (false) in both methods.   No need to change here, unless you change default for __init__

Reply:
Thanks for your review, I committed a patch to fix docstring and unit test error.
Could you please review again?